.travis.yml

```diff
@@ -33,5 +33,4 @@ jobs:
         prerelease: true
         overwrite: true
         name: master
-        body: "Changes from $TRAVIS_COMMIT containing $TRAVIS_COMMIT_MESSAGE"
```

Comment:
can you change `debug-app.apk` to `app-debug.apk` in this file? I didn't verify the file name before 😅